socorro/external/boto/connection_context.py

```diff
@@ -73,58 +72,39 @@ class ConnectionContextBase(RequiredConfig):
         reference_value_from='resource.boto'
     )
 
-    operational_exceptions = (
+    RETRYABLE_EXCEPTIONS = (
         socket.timeout,
-        # wild guesses at retriable exceptions
         boto.exception.PleaseRetryException,
         boto.exception.ResumableTransferDisposition,
         boto.exception.ResumableUploadException,
     )
 
-    conditional_exceptions = (
-        boto.exception.StorageResponseError
-    )
-
-    def is_operational_exception(self, x):
-        if "not found, no value returned" in str(x):
-            # the not found error needs to be re-tryable to compensate for
-            # eventual consistency.  However, a method capable of raising this
-            # exception should never be used with a transaction executor that
-            # has infinite back off.
```

Comment:
We no longer do infinite backoffs. This is in the crontabber and processor, so if they fail in some long, but ephemeral issue, that's "ok" and we can recover from it. We don't want to wait infinitely.